doc/release-notes.md

```diff
@@ -4,6 +4,34 @@ release-notes at release time)
 Notable changes
 ===============
 
+The mainnet activation of the Canopy network upgrade is supported by this
+release, with an activation height of 1046400, which should occur roughly in the
+middle of November — following the targeted EOS halt of our 3.1.0 release.
+Please upgrade to this release, or any subsequent release, in order to follow
+the Canopy network upgrade.
+
+The following ZIPs are being deployed as part of this upgrade:
+
+* [ZIP 207: Funding Streams](https://zips.z.cash/zip-0207) in conjunction with [ZIP 214: Consensus rules for a Zcash Development Fund](https://zips.z.cash/zip-0214)
+* [ZIP 211: Disabling Addition of New Value to the Sprout Value Pool](https://zips.z.cash/zip-0211)
+* [ZIP 212: Allow Recipient to Derive Sapling Ephemeral Secret from Note Plaintext](https://zips.z.cash/zip-0212)
+* [ZIP 215: Explicitly Defining and Modifying Ed25519 Validation Rules](https://zips.z.cash/zip-0215)
+
+In order to help the ecosystem prepare for the mainnet activiation, Canopy has
+already been activated on the Zcash testnet. Any node version 3.1.0 or higher,
+including this release, supports the Canopy activation on testnet.
+
+Disabling new value in the Sprout value pool
+--------------------------------------------
+
+After the mainnet activation of Canopy, it will not be possible to send funds to
+Sprout z-addresses from any _other_ kind of address, as described in [ZIP 211](https://zips.z.cash/zip-0211).
+It will still be possible to send funds _from_ a Sprout z-address and to send
+funds between Sprout addresses. Users of Sprout z-addresses are encouraged to
```

Comment:
[ZIP 212](https://zips.z.cash/zip-0211#specification) currently says:
> When this proposal is activated, nodes and wallets MUST disable any facilities to send to Sprout addresses, and this SHOULD be made clear in user interfaces and API documentation.

NCC raised this as an ambiguity in the ZIP:

![image](https://user-images.githubusercontent.com/643204/91363732-257a5780-e7f5-11ea-8d5c-c8e30ab2081a.png)

If the intent is that fully shielded Sprout->Sprout and/or partially shielded Sprout -> Sprout+{transparent, Sapling} transactions are allowed, I can change the ZIP to say that. We need to check what is actually allowed by the wallet code.

Reply:
I will fix the ZIP, but the release notes look correct here.